.gitlab-ci.yml

```diff
@@ -48,7 +49,7 @@ before_script:
 
     - echo 'start:coq.build'
     - make -j "$NJOBS" byte
-    - make -j "$NJOBS"
+    - make -j "$NJOBS" world $EXTRA_TARGET
```

Comment:
Psss, I am not happy to see such hacks, in fact, `coq_config.py` should have never been introduced in the first place like it is now [a configure target] but instead it should have been generated using a make rule [as the only thing it contains is the version]. But when I saw it I didn't want to complain as they doc people had enough of a party back then.